docs/02-concepts/03-cryptography/00-commitment_scheme.md

```diff
@@ -0,0 +1,36 @@
+# Commitment Scheme
+
+
+## Binding and hiding qualities 
+
+A cryptographic commitment scheme is a method to commit to a secret value $m$, without revealing it. A commitment scheme must be both **binding** and **hiding**.
+
+**Binding**: Given a commitment $C = commit(m)$, it is not possible to forge the commitment $C$ with another value $m'$ such that $m' \neq m$ and $C = commit(m) = commit(m')$.
+
+**Hiding**: Given a commitment $C$, where $C = commit(m)$, it is not possible to discover $m$.
+
+An analogy is writing a secret password on a piece of paper, placing it in a safe, and handing it to a friend. Without opening the safe, your friend cannot determine the password - this is the hiding property. Once you have given the safe away you can no longer change the password - this is the binding property. You have committed to the password using the safe, and you can later reveal the password by opening the safe.
+
+## Prover and verifier
+
+Commitment schemes involve a **prover** and a **verifier**. The prover has the secret value $m$ known only to themelves, and creates a commitment $C$ which they share with the verifier. At a later time the prover can share more information (including the secret value $m$) with the verifier to 'open' or 'reveal' the commitment. The verifier can then verify that $m$ was the value used to generate $C$.
```

Comment:
Minor note that we don't need to change; we usually specify a committer and an opener/receiver when talking about commitment schemes